src/pages/RFOX/components/Claim/ClaimConfirm.tsx

```diff
@@ -0,0 +1,138 @@
+import { Timeline, TimelineItem } from 'components/Timeline/Timeline'
+import { fromBaseUnit } from 'lib/math'
+import { selectAssetById } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { ClaimRoutePaths, type ClaimRouteProps, type RfoxClaimQuote } from './types'
+
+type ClaimConfirmProps = {
+  claimQuote: RfoxClaimQuote
+  setClaimTxid: (txId: string) => void
+}
+
+const backIcon = <ArrowBackIcon />
+
+const CustomRow: React.FC<RowProps> = props => <Row fontSize='sm' fontWeight='medium' {...props} />
+
+export const ClaimConfirm: FC<ClaimRouteProps & ClaimConfirmProps> = ({
+  claimQuote,
+  setClaimTxid,
+}) => {
+  const history = useHistory()
+  const translate = useTranslate()
+
+  const handleGoBack = useCallback(() => {
+    history.push(ClaimRoutePaths.Select)
+  }, [history])
+
+  const claimAsset = useAppSelector(state => selectAssetById(state, claimQuote.claimAssetId))
+  const claimAmountCryptoPrecision = useMemo(
+    () => fromBaseUnit(claimQuote.claimAmountCryptoBaseUnit, claimAsset?.precision ?? 0),
+    [claimQuote.claimAmountCryptoBaseUnit, claimAsset?.precision],
+  )
+
+  const claimCard = useMemo(() => {
+    if (!claimAsset) return null
+    return (
+      <Card
+        display='flex'
+        alignItems='center'
+        justifyContent='center'
+        flexDir='column'
+        gap={4}
+        py={6}
+        px={4}
+        flex={1}
+        mx={-2}
+      >
+        <AssetIcon size='sm' assetId={claimAsset?.assetId} />
+        <Stack textAlign='center' spacing={0}>
+          <Amount.Crypto value={claimAmountCryptoPrecision} symbol={claimAsset?.symbol} />
+          <Amount.Fiat fontSize='sm' color='text.subtle' value='10.22' />
+        </Stack>
+      </Card>
+    )
+  }, [claimAsset, claimAmountCryptoPrecision])
+
+  const handleSubmit = useCallback(() => {
+    setClaimTxid('1234')
+    history.push(ClaimRoutePaths.Status)
+  }, [history, setClaimTxid])
+
+  return (
+    <SlideTransition>
+      <CardHeader display='flex' alignItems='center' gap={2}>
+        <Flex flex={1}>
+          <IconButton onClick={handleGoBack} variant='ghost' aria-label='back' icon={backIcon} />
+        </Flex>
+        <Flex textAlign='center'>{translate('common.confirm')}</Flex>
+        <Flex flex={1}></Flex>
+      </CardHeader>
+      <CardBody>
+        <Stack spacing={6}>
+          {claimCard}
+          <Timeline>
+            <TimelineItem>
+              <CustomRow>
+                <Row.Label>{translate('RFOX.claimReceiveAddress')}</Row.Label>
+                <Row.Value>0xSomething</Row.Value>
+              </CustomRow>
+            </TimelineItem>
+            <TimelineItem>
+              <CustomRow>
+                <Row.Label>{translate('RFOX.networkFee')}</Row.Label>
+                <Row.Value>
+                  <Skeleton isLoaded={true}>
+                    <Row.Value>
+                      <Amount.Fiat value={'1.23' ?? '0.0'} />
```

Comment:
🤔

Reply:
Actioned in https://github.com/shapeshift/web/pull/7004/commits/b39ea0eb699d2309beeb8cef365131bd0bba9bab